compiler/rustc_macros/src/lib.rs

```diff
@@ -19,9 +19,10 @@ pub fn rustc_queries(input: TokenStream) -> TokenStream {
     query::rustc_queries(input)
 }
 
+#[cfg(not(test))]
 #[proc_macro]
 pub fn symbols(input: TokenStream) -> TokenStream {
-    symbols::symbols(input)
+    symbols::symbols(input.into()).into()
```

Comment:
```suggestion
    symbols::symbols(input)
```

Reply:
These `into()` calls are necessary in order to convert between `proc_macro2::TokenStream` and `proc_macro::TokenStream`. When compiling as a proc-macro, this is a no-op.